airflow/bin/cli.py

```diff
@@ -884,13 +884,13 @@ def webserver(args):
         print(
             "Starting the web server on port {0} and host {1}.".format(
                 args.port, args.hostname))
-        app, _ = create_app(conf, testing=conf.get('core', 'unit_test_mode'))
+        app, _ = create_app(None, testing=conf.get('core', 'unit_test_mode'))
```

Comment:
I am not sure if I follow, but why do we change the conf to None here?

Reply:
Previously the `config` argument to create_app wasn't used by that app, but in tidying up of the tests I started using it -- it is now a dict that is merged in to the flask `app.config`.

Reply:
Previously the `config` argument to create_app wasn't used by that app, but in tidying up of the tests I started using it -- it is now a dict that is merged in to the flask `app.config`.

https://github.com/apache/airflow/pull/4910/files#diff-3d86925e55c01b3d8f3cb10def92fd1cR62

Reply:
Got it. thanks.